apps/crn-server/src/data-providers/contentful/users.data-provider.ts

```diff
@@ -340,3 +391,75 @@ const parseOrcidWorksContentful = (
     throw new Error(`Invalid ORCID works content data: ${e}`);
   }
 };
+
+const parseToWorkingGroups = (
+  users: (GroupMemberItem | GroupLeaderItem)[],
+  isAlumni: boolean,
+): WorkingGroupMembership[] =>
+  users.reduce(
+    (
+      workingGroups: WorkingGroupMembership[],
+      user: GroupMemberItem | GroupLeaderItem,
+    ) => {
+      const workingGroup = user.linkedFrom?.workingGroupsCollection?.items[0];
+      const isInActive = !!user.inactiveSinceDate;
+
+      workingGroups.push({
+        id: workingGroup?.sys.id,
+        name: workingGroup?.title,
+        role: (user as GroupLeaderItem).role || 'Member',
+        active: workingGroup?.complete ? false : !isAlumni && !isInActive,
+      } as WorkingGroupMembership);
+
+      return workingGroups;
+    },
+    [],
+  );
+
+const parseToInterestGroups = (teams: TeamMembership[]) =>
```

Comment:
I would prefer to see the return types on these functions (same as above) - reason being - I have a feeling TypeScript might often infer it to something that is not quite what we are looking for and then when spreading that onto the end object we might never realise it was invalid